plugins/meta/portmap/chain.go

```diff
@@ -35,16 +35,17 @@ type chain struct {
 
 // setup idempotently creates the chain. It will not error if the chain exists.
 func (c *chain) setup(ipt *iptables.IPTables) error {
-	// create the chain
-	exists, err := chainExists(ipt, c.table, c.name)
```

Comment:
I think `get` is cheaper than `create`, so maybe it's better to keep these lines...

Reply:
Sure, will do.

Reply:
Done